.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,81 @@
+---
+name: "\U0001F41B Bug Report"
+about: "Please report issues related to the TypeScript and HTML API of Vaadin components here."
+title: ''
+labels: 'i: needs triage'
+assignees: ''
+
+---
+
+<!--
+  Please fill out each section below, otherwise, your issue risks taking an infinite amount of time to triage. This info allows the maintainers team to diagnose and fix your issue as quickly as possible.
+
+  Before opening a new issue, please search existing issues: https://github.com/vaadin/web-components/issues.
+-->
+
+### Description
+
+Describe the issue that you're seeing.
+
+<!-- Example: The `vaadin-grid` element scrolls when applying filter. -->
+
+### Expected outcome
+
+What should happen?
+
+<!-- Example: No scrolling should happen. -->
+
+### Actual outcome
+
+What happened.
+
+<!-- Example: Grid content occasionally gets scrolled. -->
+
+### Live Demo
+
+A screenshot or video with issue reproduced.
+
+### Minimal reproducible example
+
+**Please** provide a code snippet with issue reproduction, this makes your issue _much_ easier to diagnose.
+
+<!--
+```
+<vaadin-grid</vaadin-grid>
+```
+```
+const grid = document.querySelector('vaadin-grid');
+grid.items = items;
```

Comment:
Can we improve this to add a filter here to make the example consistent with the remaining examples?

Reply:
Adding `vaadin-grid-filter` would probably explode this example with too many details. Here I am trying to show how users can provide a code snippet rather than provide an explicit one.

If you think a filter element is absolutely needed, I can add it.

Reply:
It's OK, I wasn't sure how much code it would add.